app/src/menu.js

```diff
@@ -1,140 +1,141 @@
-          };
-          electron.dialog.showMessageBox(focusedWindow, options, () => {});
-        }
+const menu = {
+  build: (electron) => {
+    const template = menu.buildTemplate(electron);
+    if (process.platform === 'darwin') {
+      addAboutMenuForMac({ template, name: electron.app.getName() });
+    } else {
+      addAboutMenuForNonMac({ template, electron });
+    }
+    return electron.Menu.buildFromTemplate(template);
+  },
+  onClickLink: (electron, url) => {
+    electron.shell.openExternal(url);
+  },
+  buildTemplate: electron =>
+    ([
+      {
+        label: i18n.t('Edit'),
+        submenu: [
+          {
+            role: 'undo',
+            label: i18n.t('Undo'),
+          },
+          {
+            role: 'redo',
+            label: i18n.t('Redo'),
+          },
+          {
+            type: 'separator',
+          },
+          {
+            role: 'cut',
+            label: i18n.t('Cut'),
+          },
+          {
+            role: 'copy',
+            label: i18n.t('Copy'),
+          },
+          {
+            role: 'paste',
+            label: i18n.t('Paste'),
+          },
+          {
+            role: 'selectall',
+            label: i18n.t('Select all'),
+          },
+        ],
       },
-    });
-  }
-  return Menu.buildFromTemplate(template);
+      {
+        label: i18n.t('View'),
+        submenu: [
+          {
+            role: 'reload',
+            label: i18n.t('Reload'),
+          },
+          {
+            role: 'togglefullscreen',
+            label: i18n.t('Toggle full screen'),
+          },
+        ],
+      },
+      {
+        label: i18n.t('Window'),
+        submenu: [
+          {
+            role: 'minimize',
+            label: i18n.t('Minimize'),
+          },
+        ],
+      },
+      {
+        label: i18n.t('Help'),
+        submenu: [
+          {
+            label: i18n.t('Lisk Website'),
+            click: menu.onClickLink.bind(null, electron, 'https://lisk.io'),
+          },
+          {
+            label: i18n.t('Lisk Chat'),
+            click: menu.onClickLink.bind(null, electron, 'https://lisk.chat'),
+          },
+          {
+            label: i18n.t('Lisk Explorer'),
+            click: menu.onClickLink.bind(null, electron, 'https://explorer.lisk.io'),
+          },
+          {
+            label: i18n.t('Lisk Forum'),
+            click: menu.onClickLink.bind(null, electron, 'https://forum.lisk.io'),
+          },
+          {
+            type: 'separator',
+          },
+          {
+            label: i18n.t('Report Issue...'),
+            click: menu.onClickLink.bind(null, electron, 'https://lisk.zendesk.com/hc/en-us/requests/new'),
+          },
+          {
+            label: i18n.t('What\'s New...'),
+            click: menu.onClickLink.bind(null, electron, 'https://github.com/LiskHQ/lisk-nano/releases'),
+          },
+        ],
+      },
+    ]),
+
 };
+
+module.exports = menu;
```

Comment:
Please use es6 export instead of 'module.export'

Reply:
done